### PR TITLE
only set default crashlytics_path value for ios

### PR DESCRIPTION
**Pull request by TaopaiC**
I have a react-native project for both android and ios app.
The default value of crashlytics_path always got "ios/Crashlytics.framework" even if I build for android platform.
Move the default value to generate_ios_command since it should only run for ios platform.

### CODE REVIEW

File: fastlane/lib/fastlane/actions/crashlytics.rb [hunk 1]
```diff
@@ -64,7 +66,6 @@ def self.available_options
           FastlaneCore::ConfigItem.new(key: :crashlytics_path,
                                        env_name: "CRASHLYTICS_FRAMEWORK_PATH",
                                        description: "Path to the submit binary in the Crashlytics bundle (iOS) or `crashlytics-devtools.jar` file (Android)",
-                                       default_value: Dir["./Pods/iOS/Crashlytics/Crashlytics.framework"].last || Dir["./**/Crashlytics.framework"].last,
```
Comment (KrauseFx) [inline fastlane/lib/fastlane/actions/crashlytics.rb]: Why did you move this over to `crashlytics_helper.rb`? Isn't this basically the same thing then?

File: fastlane/lib/fastlane/actions/crashlytics.rb [hunk 2]
```diff
@@ -55,7 +57,7 @@ def self.available_options
           FastlaneCore::ConfigItem.new(key: :apk_path,
                                        env_name: "CRASHLYTICS_APK_PATH",
                                        description: "Path to your APK file",
-                                       default_value: Actions.lane_context[SharedValues::GRADLE_APK_OUTPUT_PATH] || Dir["*.apk"].last || Dir[File.join("app", "build", "outputs", "apk", "app-Release.apk")].last,
+                                       default_value: Actions.lane_context[SharedValues::GRADLE_APK_OUTPUT_PATH] || platform == :android && Dir[File.join("app", "build", "outputs", "apk", "app-release.apk")].last,
```
Comment (KrauseFx) [inline fastlane/lib/fastlane/actions/crashlytics.rb]: This line is too long now, can we also move this up to line 44 instead?

File: fastlane/lib/fastlane/actions/crashlytics.rb [hunk 3]
```diff
@@ -41,12 +41,14 @@ def self.description
...
 
       def self.available_options
+        platform = Actions.lane_context[Actions::SharedValues::PLATFORM_NAME]
+
         [
           # iOS Specific
           FastlaneCore::ConfigItem.new(key: :ipa_path,
                                        env_name: "CRASHLYTICS_IPA_PATH",
                                        description: "Path to your IPA file. Optional if you use the `gym` or `xcodebuild` action",
-                                       default_value: Actions.lane_context[SharedValues::IPA_OUTPUT_PATH] || Dir["*.ipa"].last,
```
Comment (KrauseFx) [inline fastlane/lib/fastlane/actions/crashlytics.rb]: This line is too long now, can we also move this up to line 44 instead?

**Comment by KrauseFx** [comment]
Related to issue https://github.com/fastlane/fastlane/issues/4430

File: fastlane/lib/fastlane/actions/crashlytics.rb (hunk 1 above)
Comment (TaopaiC) [inline fastlane/lib/fastlane/actions/crashlytics.rb]: It always set to the "Crashlytics.framework" directory if the directory existed, even if I build for android platform. so I move it to `generate_ios_command`.

File: fastlane/lib/fastlane/actions/crashlytics.rb [hunk 4]
```diff
@@ -41,12 +41,16 @@ def self.description
       end
 
       def self.available_options
+        platform = Actions.lane_context[Actions::SharedValues::PLATFORM_NAME]
+        ipa_path_default = (!platform || platform == :ios) && Dir["*.ipa"].last
```
Comment (KrauseFx) [inline fastlane/lib/fastlane/actions/crashlytics.rb]: Can we move this to multiple lines? For non-Ruby developers the `&&` operator may not be 100% clear

Comment (KrauseFx) [inline fastlane/lib/fastlane/actions/crashlytics.rb]: Something like 
```ruby
ipa_path_default = nil
if platform == :ios or platform.nil?
   ipa_path_default = Dir["*.ipa"].last
end
```
and the same for Android 👍

Comment (TaopaiC) [inline fastlane/lib/fastlane/actions/crashlytics.rb]: changed on both `ipa_path_default` and `apk_path_default`

File: fastlane/lib/fastlane/actions/crashlytics.rb (hunk 1 above)
Comment (KrauseFx) [inline fastlane/lib/fastlane/actions/crashlytics.rb]: Oh right, that makes sense :+1:

**Comment by KrauseFx** [comment]
Thanks for your contribution
